test/unit/modules.spec.ts

```diff
@@ -41,20 +41,6 @@ export class LuaModuleTests {
         Expect(lua).toBe(`require("lualib_bundle")`);
     }
 
-    @Test("Import named bindings exception")
-    public namedBindigsException(): void {
```

Comment:
Maybe ignore this for now instead of removing, since we probably want to still have this test on the transformer instead of transpiler.

Reply:
This test was using a custom created AST/Node I don't think that named bindings exception can ever be thrown.